INSTALLATION.md

```diff
@@ -1,76 +1,76 @@
+
 # Installation
 
 * [Introduction](#introduction)
-* [Step by step instructions](#step-by-step-instructions)
+* [Desktop](#desktop)
   * [MacOS](#macos)
   * [Linux](#linux)
   * [Windows](#windows)
 
 ## Introduction
 
-This document will guide you through the installation process of the latest tagged version of Thyra.
-
-> **_PREREQUISITES:_** Be comfortable with your terminal and have a recent version of MacOS, Windows or Linux.
-
-> **_TROUBLESHOOTING:_** If you have trouble following this procedure, feel free to [open a question](https://github.com/massalabs/thyra/issues/new) describing your problem.
+From this point you'll start to custom your Massa's journey thanks to Thyra. If you want to learn more about Thyra's objectives and mindset, go here; if you're curious about the technical aspects and want to contribute go [there](./CONTRIBUTING.md).
 
-## Step by step instructions
+This document will guide you through the installation process and the first steps to get started with Thyra.
 
-Two steps are required to use our web on-chain product:
 
-* Thyra installation : obtain the Thyra binary corresponding to your operating system (OS), rename it and make it executable.
-* DNS configuration : install and configure your DNS to resolve the massa top level domain (*.massa) where Thyra runs.
+> **_TROUBLESHOOTING:_** If you have trouble following this procedure, feel free to [open a question](https://github.com/massalabs/thyra/issues/new) describing your problem.
 
-The following installation guide allows you to perform these steps automatically using a simple script to run on your terminal.
+## Instructions by Devices
 
-Now, let's move on to your OS section:
+By following these instructions, you will: 
+* Install Thyra to access and perform actions on the blockchain. 
+* Install Thyra application to interact and use Thyra and its plugins very easily.
 
-* [Linux](#linux)
-* [Windows](#windows)
-* [MacOS](#macos)
+### Desktop
 
+#### MacOS
 
-### MacOS
+1. Check out your Mac CPU to know which link above you should use.
+* If you have an Apple Silicon CPUs (M1, M2, ...), download ARM installer [here](https://github.com/massalabs/thyra/releases/download/v0.1.12/thyra-installer_darwin_arm64.tar.gz)
+* If you have an Intel CPUs (i5, i7, ...), download ADM installer [here](https://github.com/massalabs/thyra/releases/download/v0.1.12/thyra-installer_darwin_amd64.tar.gz)
```

Comment:
Since it's `a` CPU, there is no need for those `s`'s:
```suggestion
* If you have an Apple Silicon CPU (M1, M2, ...), download ARM installer [here](https://github.com/massalabs/thyra/releases/download/v0.1.12/thyra-installer_darwin_arm64.tar.gz)
* If you have an Intel CPU (i5, i7, ...), download ADM installer [here](https://github.com/massalabs/thyra/releases/download/v0.1.12/thyra-installer_darwin_amd64.tar.gz)
```